docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -4390,6 +4390,11 @@ tsdb:
   # CLI flag: -blocks-storage.tsdb.head-chunks-write-queue-size
   [head_chunks_write_queue_size: <int> | default = 1000000]
 
+  # (advanced) When enabled, ingester will produce 24h blocks for the out of
```

Comment:
Usually we start with `experimental` if we're not sure the option is going to stick around. Are we planning on keeping this option forever?

Reply:
> Are we planning on keeping this option forever?

Most likely, but we can start with experimental for now